scene/gui/rich_text_label.cpp

```diff
@@ -588,7 +588,7 @@ void RichTextLabel::_shape_line(ItemFrame *p_frame, int p_line, const Ref<Font>
 					offset.x += table->columns[column].width + hseparation + frame->padding.size.x;
 
 					row_height = MAX(yofs, row_height);
-					if (column == col_count - 1) {
+					if (column == col_count - 1 || E->next() == nullptr) {
```

Comment:
Maybe we can add a comment to make the meaning of the code more obvious?
E.g.
`// Wrap the row on last column or the last cell.`

Reply:
Note sure of the meaning of "wrap" here.

I've added `// Add row height after last column of the row or last cell of the table.`